templates/slurm_stats.sh.epp

```diff
@@ -28,10 +28,14 @@ echo "slurm_nodesumdata,partition=all,type=total count=${nodes_total}"
 total_jobs=$(cat $tf | wc -l)
 running_jobs=$(cat $tf | grep RUNNING | wc -l )
 pending_jobs=$(cat $tf | grep PENDING | wc -l )
+max_jobs=$("${slurm_path}"/scontrol show config | grep MaxJobCount | cut -d' ' -f 15)
+percent_max=$(echo "${total_jobs} / ${max_jobs} *100" | bc -l)
 
 echo "slurm_jobsumdata,partition=all,type=running count=${running_jobs}"
 echo "slurm_jobsumdata,partition=all,type=pending count=${pending_jobs}"
 echo "slurm_jobsumdata,partition=all,type=total count=${total_jobs}"
+echo "slurm_jobsumdata,partition=all type=max_jobs count=${max_jobs}"
+echo "slurm_jobsumdata,partition=all type=percent_of_max_jobs count=${percent_max}"
```

Comment:
I think these are missing comma before the `type=`.

Should these be:
```suggestion
echo "slurm_jobsumdata,partition=all,type=max_jobs count=${max_jobs}"
echo "slurm_jobsumdata,partition=all,type=percent_of_max_jobs count=${percent_max}"
```

Reply:
@billglick Yes good catch